docs/Tutorial_Advanced_Annotation_Vectors.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>Instead of "It summarizes...", I would just combine it with the previous sentence with `and it summarizes..."</p><p> </p><p>"and  replicates THE presented case studies"</p><p> </p><p>Instead of "In the first tutorial", I think its better to say " In the Basics of Annotation Vectors tutorial, we..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>